### DISCUSSION TITLE
Don't shorten the repository names on personal page

### DISCUSSION THREAD

**Discussion by WhiteBlackGoose** [community/community#18763] [Other Feature Feedback, Questions, & Ideas]
It's not like there's not enough space to fit in the title

![image](https://user-images.githubusercontent.com/31178401/174150097-9255e7a8-bcaa-46fe-965f-463781e7eb8d.png)

**Comment by Andyveloper** [comment]
Couldn't agree more

**Comment by sen-den** [comment]
The same issue is presented in Safari

**Comment by Jesuis33** [comment]
https://github.com/platformio/platformio-core-installer/issues/2193#issue-1274107917

**Comment by jofftiquez** [comment]
This is a reasonable request
